.github/workflows/create_release.yaml

```diff
@@ -0,0 +1,49 @@
+name: Release a new version of the plugin
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+    create-release:
+        name: Create Github release
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4.1.1
+            - name: Set up Python 3.10
+              uses: actions/setup-python@v5
+              with:
+                python-version: "3.10"
+
+            - name: Install Poetry
+              run: |
+                curl -sSL https://install.python-poetry.org | python - -y
+                echo "$HOME/.local/bin" >> $GITHUB_PATH
+
+            - name: Bump Version
+              id: bump-version
+              run: |
+                # Bump minor version with poetry
+                poetry version minor
+                VERSION=$(poetry version --short)
+                echo VERSION=${VERSION} >> $GITHUB_ENV
+
+                # Tag the commit with the new version
+                git tag $VERSION
+                git push origin $VERSION
+
+                # Push version update in pyproject.toml
+                git config --global user.name 'IS DevOps Bot'
+                git config --global user.email 'is-devops-bot@users.noreply.github.com'
+                git add .
```

Comment:
Is it safe to add all files, or are there some that are better ignored? poetry would create a lock file e.g., which might be fine to add.

Reply:
poetry.lock won't be generated here but I changed the command to only commit pyproject.toml